projectile.el

```diff
@@ -1489,36 +1489,45 @@ Never use on many files since it's going to recalculate the
 project-root for every file."
   (expand-file-name name (projectile-project-root)))
 
-(defun projectile-completing-read (prompt choices &optional initial-input)
+(cl-defun projectile-completing-read (prompt choices
```

Comment:
Do we really need to use `cl-defun` here? I get why you did it, but I generally dislike introducing interfaces that are not common for Elisp.